docs/core/tutorials/netcore-hosting.md

```diff
@@ -46,51 +46,51 @@ Once found, the library is loaded with `LoadLibraryEx` (on Windows) or `dlopen`
+### Step 3 - Prepare runtime properties
 
-Before starting the runtime, we need to prepare properties that will specify AppDomain behavior (especially concerning the assembly loader). The properties are an array of keys and a corresponding array of values that are used when starting the runtime and default AppDomain.
+Before starting the runtime, it is necessary to prepare some properties to specify behavior (especially concerning the assembly loader). 
 
-Common AppDomain properties include:
+Common properties include:
 
-* `TRUSTED_PLATFORM_ASSEMBLIES` This is a list of assembly paths (delimited by ';' on Windows and ':' on Linux) which the AppDomain should prioritize loading and give full trust to (even in partially-trusted domains). This list is meant to contain 'Framework' assemblies and other platform assemblies, similar to the global assembly cache (GAC) in .NET Framework scenarios. Some hosts will put any library next to *coreclr.dll* on this list, others have hard-coded manifests listing trusted assemblies for their purposes.
+* `TRUSTED_PLATFORM_ASSEMBLIES` This is a list of assembly paths (delimited by ';' on Windows and ':' on Linux) which the runtime will be able to resovle by default. Some hosts have hard-coded manifests listing assemblies they can load. Others will put any library in certain locations (next to *coreclr.dll*, for example) on this list.
 * `APP_PATHS` This is a list of paths to probe in for an assembly if it can't be found in the trusted platform assemblies (TPA) list. These paths are meant to be the locations where users' assemblies can be found. Common values for this property include the path the target app was loaded from or other locations where user assets are known to live.
-*  `APP_NI_PATHS` This list is very similar to APP_PATHS except that it's meant to be paths that will be probed for native images.
-*  `NATIVE_DLL_SEARCH_DIRECTORIES` This property is a list of paths the loader should probe when looking for native DLLs called via p/invoke.
+*  `APP_NI_PATHS` This list is similar to APP_PATHS except that it's meant to be paths that will be probed for native images.
+*  `NATIVE_DLL_SEARCH_DIRECTORIES` This property is a list of paths the loader should probe when looking for native libraries called via p/invoke.
 *  `PLATFORM_RESOURCE_ROOTS` This list includes paths to probe in for resource satellite assemblies (in culture-specific sub-directories).
 
-In our sample host, the TPA list is constructed by simply listing all libraries in the current directory:
+In this sample host, the TPA list is constructed by simply listing all libraries in the current directory:
 
 [!code-cpp[CoreClrHost#7](../../../samples/core/hosting/HostWithCoreClrHost/SampleHost.cpp#7)]
 
-Because the sample is simple, we only need the `TRUSTED_PLATFORM_ASSEMBLIES` and `APP_PATHS` properties:
+Because the sample is simple, it only needs the `TRUSTED_PLATFORM_ASSEMBLIES` and `APP_PATHS` properties:
```

Comment:
As stated above, it doesn't need the APP_PATHS, we should remove the mention here.